sdk/eventhub/azure-eventhubs/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -0,0 +1,187 @@
+    ) -> None:
+        """Receive events from partition(s) optionally with load balancing and checkpointing.
+
+        :param on_event: The callback function that process received events
+        :param consumer_group: Receive events from the event hub for this consumer group
+        :param partition_id: Receive from this partition only if it's not None. Receive from all partition otherwise.
+        :param owner_level: The priority of the exclusive consumer. An exclusive
+         consumer will be created if owner_level is set. Higher owner_level has higher exclusive priority.
+        :param prefetch: The number of events to prefetch from the service
+         for processing. Default is 300.
+        :param track_last_enqueued_event_properties: Indicates whether or not the consumer should request information
+         on the last enqueued event on its associated partition, and track that information as events are received.
+         When information about the partition's last enqueued event is being tracked, each event received from the
+         Event Hubs service will carry metadata about the partition. This results in a small amount of additional
+         network bandwidth consumption that is generally a favorable trade-off when considered against periodically
+         making requests for partition properties using the Event Hub client.
+         It is set to `False` by default.
+        :param initial_event_position: Start receiving from this initial_event_position
+         if there isn't checkpoint data for a partition. Use the checkpoint data if there it's available. This can be a
+         a dict with partition id as the key and position as the value for individual partitions, or a single
+         EventPosition instance for all partitions.
+        :param on_error: The callback function that is called when there is error during receiving events.
+        :param on_partition_initialize: The callback function that is called right before
+         a partition is initialized.
+        :param on_partition_close: The callback function that is called when this receive method finishes receiving
+         from a partition.
+        :rtype: None
+
+        Example:
+            .. literalinclude:: ../samples/test_examples_eventhub_async.py
+                :start-after: [START eventhub_consumer_client_receive_async]
+                :end-before: [END eventhub_consumer_client_receive_async]
+                :language: python
+                :dedent: 4
+                :caption: Receive events from the EventHub.
+        """
+        async with self._lock:
+            error = None
+            if (consumer_group, '-1') in self._event_processors:
+                error = ValueError("This consumer client is already receiving events from all partitions for"
+                                 " consumer group {}. "
+                                 "Shouldn't receive from any other partitions again".format(consumer_group))
```

Comment:
The last sentence does not add any additional information. Please remove.

Reply:
Removed